mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
```

Comment:
The function name `removeOrder` is not following the conventional naming style of the other functions in the file, which are in camelCase. It should be renamed to `removeOrderById` for consistency.

The function is not validating the `req.params.orderId` before using it to delete the order. It should be validated to ensure it's a valid ObjectId.

The function is not handling the case where `req.params.orderId` is not provided. It should return a 400 error response with a meaningful error message in this case.

The error message "Order not found" is not very informative. It should be more descriptive, such as "Order with ID {req.params.orderId} not found".

The success message "Order removed successfully" is not very informative. It should include the ID of the removed order, such as "Order with ID {req.params.orderId} removed successfully".